<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout" rowstretch="9,0" columnstretch="1,0,0">
    <item row="1" column="0" colspan="2">
     <widget class="QPlainTextEdit" name="output_log">
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QListWidget" name="imageList">
      <property name="iconSize">
       <size>
        <width>200</width>
        <height>150</height>
       </size>
      </property>
      <property name="movement">
       <enum>QListView::Movement::Static</enum>
      </property>
      <property name="flow">
       <enum>QListView::Flow::TopToBottom</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>true</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::ResizeMode::Adjust</enum>
      </property>
      <property name="viewMode">
       <enum>QListView::ViewMode::IconMode</enum>
      </property>
      <property name="itemAlignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="rightPanel" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="HistViewer" name="hist" native="true"/>
       </item>
       <item>
        <widget class="FileInfoViewer" name="info" native="true"/>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QMdiArea" name="mdiArea">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
      <property name="horizontalScrollBarPolicy">
       <enum>Qt::ScrollBarPolicy::ScrollBarAsNeeded</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1920</width>
     <height>33</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenFile"/>
    <addaction name="actionOpenFolder"/>
   </widget>
   <widget class="QMenu" name="menuTheme">
    <property name="title">
     <string>Theme</string>
    </property>
    <addaction name="actionDark"/>
    <addaction name="actionLight"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionSystemInfo"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTheme"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenFile">
   <property name="icon">
    <iconset>
     <normaloff>:/ImageTool/UI_Icons/open.png</normaloff>:/ImageTool/UI_Icons/open.png</iconset>
   </property>
   <property name="text">
    <string>OpenFile</string>
   </property>
  </action>
  <action name="actionDark">
   <property name="text">
    <string>Dark</string>
   </property>
  </action>
  <action name="actionLight">
   <property name="text">
    <string>Light</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionSystemInfo">
   <property name="text">
    <string>SystemInfo</string>
   </property>
  </action>
  <action name="actionOpenFolder">
   <property name="text">
    <string>OpenFolder</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>HistViewer</class>
   <extends>QWidget</extends>
   <header>histviewer.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FileInfoViewer</class>
   <extends>QWidget</extends>
   <header>fileinfoviewer.hpp</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
